script/release-publish.sh

```diff
@@ -47,6 +47,12 @@ ZIPF_NO_VER="target/universal/apalache.zip"
 TGZF="target/universal/apalache-${VERSION}.tgz"
 TGZF_NO_VER="target/universal/apalache.tgz"
 
+cd target/universal
+sha256sum apalache.* > sha256sum.txt
+cd -
+
+SHA256F="target/universal/sha256sum.txt"
+
```

Comment:
Could this be simplified a bit by using the variables defined so dir changes aren't needed? E.g.,

```suggestion
SHA256F="target/universal/sha256sum.txt"
sha256sum "$ZIPF_NO_VER" "$TGZF_NO_VER" > "$SHA256F"
```

What do you think?

Reply:
I avoided doing this because the output will include the given path, not just `apalache.tgz` or `apalache.zip`.

```
$ sha256sum target/universal/apalache.zip target/universal/apalache.tgz 
7dd22045cea14cedbfccfc48f7a3e09233126f270f34f4e5f6667e55622e286d  target/universal/apalache.zip
c18b054f228a4bd7e3a96960b316c04d3255274c8cdf921a136d2b7e37062685  target/universal/apalache.tgz
```

Do you have any other ideas to avoid this?

Reply:
Ah, I see. I'm OK with keeping it as is then.

Reply:
Makes sense, but should we also report hashes of the the versioned bundles?

Reply:
I'd still clean up the shafile path by moving the declaration of `SHA256F` to the top as suggested by Shon. Perhaps running in a subshell is the most concise:

```suggestion
SHA256F="target/universal/sha256sum.txt"
(cd target/universal && sha256sum apalache.* > "$SHA256F")
```

Reply:
I like this!